### PR TITLE
Fixup #1625: in MSA explicitly call setVisible(true)

### PR DESCRIPTION
**Pull request by rhaschke**
... to actually show the robot of the RobotStateDisplay (which is hidden initially since #1625).
Fixes b0f01b2dc79f99f5650933834c8e50905bdbe168.
Addresses https://github.com/ros-planning/moveit/issues/1736#issuecomment-554953490.

### CODE REVIEW

File: moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp [hunk 1]
```diff
@@ -416,7 +421,10 @@ void RobotStateDisplay::update(float wall_dt, float ros_dt)
   if (load_robot_model_)
   {
     loadRobotModel();
+    // The following call to changedRobotStateTopic() should not change visibility
+    bool visible = robot_->isVisible();
     changedRobotStateTopic();
+    robot_->setVisible(visible);
```
Comment (v4hn) [inline moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp]: Why is that needed? What might change the visibility when changing the topic?

Comment (rhaschke) [reply moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp]: `changedRobotStateTopic()` resets the robot to invisible to indicate that we didn't receive a RobotState message on the (new) topic yet. However, here we want to keep the current visibility state.
